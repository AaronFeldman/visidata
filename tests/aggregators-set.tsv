Item	avg_Total	sum_Total	sum_Unit_Cost	sum_Units
Binder	638.51	9577.65	9577.65	9577
Pencil	164.24	2135.14	2135.14	2135
Pen Set	595.70	4169.87	4169.87	4169
Pen	409.04	2045.22	2045.22	2045
Desk	566.67	1700.00	1700.00	1700
