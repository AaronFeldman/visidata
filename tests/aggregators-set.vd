sheet	col	row	keystrokes	input	comment
		0	o	sample_data/sample.tsv	open file
sample	Units	0	#		sets type of current column to int
sample	Units	0	+	sum	adds aggregator to the current column
sample	Unit_Cost	0	%		sets type of current column to float
sample	Total	0	%		sets type of current column to float
sample	Total	0	C		opens Columns Sheet
sample_columns	name	6	g+	sum	adds aggregator to selected source columns
sample_columns	aggregators	6	e	sum avg	edits contents of current cell
sample_columns	aggregators	6	q		quit current sheet
sample	Item	0	F		opens Frequency Table grouped on current column
sample_Item_freq	Item	0	C		opens Columns Sheet
sample_Item_freq_columns	name	0	[		sorts ascending by current column
sample_Item_freq_columns	value	5	,		selects rows matching current cell in current column
sample_Item_freq_columns	value	5	g-		hides selected columns on source sheet
sample_Item_freq_columns	name	5	q		quit current sheet
