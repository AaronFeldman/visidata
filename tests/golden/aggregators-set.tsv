Item	avg_Total	sum_Total	sum_Unit_Cost	sum_Units	sum_Units
Binder	638.51	9577.65	172.86	722	722
Pencil	164.24	2135.14	36.07	716	716
Pen Set	595.70	4169.87	83.39	395	395
Pen	409.04	2045.22	55.95	278	278
Desk	566.67	1700.00	525.00	10	10
